src/main/flight/pid.c

```diff
@@ -1343,7 +1351,13 @@ void FAST_CODE pidController(const pidProfile_t *pidProfile, timeUs_t currentTim
         // b = 1 and only c (feedforward weight) can be tuned (amount derivative on measurement or error).
 
         // -----calculate P component
-        pidData[axis].P = pidCoefficient[axis].Kp * errorRate * tpaFactorKp;
+        // If enabled vary P and D using the content of AUXn (Heli-like vehicles, gyro-sensitivity channel).
+        float auxPgain = 1;
```

Comment:
Please use camelCase.

Reply:
@mikeller What name do you mean?

Reply:
`auxPgain`

Reply:
@mikeller: I still don't see any difference to names like `pidData` or `errorRate`.

Reply:
Do you mean sth like `auxpGain`?